src/npcmove.cpp

```diff
@@ -1669,12 +1662,12 @@ void npc::alt_attack(int target)
         std::vector<point> trajectory;
         int linet, light = g->light_level();
 
-        if (dist <= confident_range(weapon_index) && wont_hit_friend(tarx, tary, weapon_index)) {
+        if (dist <= confident_range(weapon_index) && wont_hit_friend( tar, weapon_index )) {
 
-            if (g->m.sees(posx(), posy(), tarx, tary, light, linet)) {
-                trajectory = line_to(posx(), posy(), tarx, tary, linet);
+            if (g->m.sees(posx(), posy(), tar.x, tar.y, light, linet)) {
+                trajectory = line_to(posx(), posy(), tar.x, tar.y, linet);
             } else {
-                trajectory = line_to(posx(), posy(), tarx, tary, 0);
+                trajectory = line_to(posx(), posy(), tar.x, tar.y, 0);
```

Comment:
Why doesn't this use the new behavior of `map::sees` that sets `linet` to 0 (like above in `npc::execute_action`)?

Is this behavior of `map::sees` different for the 2D and 3D version? If so, that would be confusing.